### DISCUSSION TITLE
reusing the Jacobian

### DISCUSSION THREAD

**Discussion by aaelmeli** [idaholab/moose#20684] [Q&A General] [answered]
Hi

I have a 3D steady-state elasticity problem that involves several coupled variables (6 variables). I used AD for computing the `Jacobian`. The solver is defined as follows:

```
[Executioner]
    type = Steady
    solve_type=LINEAR
    petsc_options_iname = ' -pc_type'
    petsc_options_value = 'lu'
[]
```
Now, if I have a second problem that has the same `Jacobian `but a different residual, Can I recover the `Jacobian `from the exodus file of the first problem and reuse it?

**Comment by GiudGiud** [comment, marked as answer]
Hello

We dont save the Jacobian to exodus. I doubt we save it to Checkpoint either. @friedmud might know
https://mooseframework.inl.gov/source/outputs/Checkpoint.html isnt very explicit, I might add to it once we resolve this.

Guillaume
